tests/runtime/uprobe

```diff
@@ -19,7 +19,7 @@ EXPECT uprobe:./testprogs/uprobe_test:function1
 TIMEOUT 5
 
 NAME "uprobes - list probes by pid"
-RUN bpftrace -l -p $(pidof uprobe_test)
+RUN bpftrace -l -p $(pidof uprobe_test) &> /tmp/probes_by_pid && grep -e '^uprobe' < /tmp/probes_by_pid
```

Comment:
Took me a moment to figure out that you went through all that to save the exit code of bpftrace. I wonder if `set -o pipefail; bpftrace .... | grep uprobe` works and is more readable

Reply:
Ha I guess I forgot the pipe existed. I should get more sleep...